frontend/packages/dev-console/integration-tests/support/pages/helm/controls.ts

```diff
@@ -0,0 +1,18 @@
+export const controls = {
+  dropdown: {
+    switchTo: (newOption: string) => {
+      cy.byLegacyTestID('dropdown-button')
+        .click()
+        .get('.pf-c-dropdown__menu')
+        .contains(newOption)
+        .click();
+    },
+    verifyVisibility: () => {
+      cy.byLegacyTestID('dropdown-button').should('be.visible');
+      cy.byLegacyTestID('dropdown-button')
+        .click()
+        .get('.pf-c-dropdown__menu')
+        .should('be.visible');
+    },
+  },
+};
```

Comment:
move it to support/commands folder

Reply:
created commands `dropdownSwitchTo` and `isDropdownVisible` and removed `frontend/packages/dev-console/integration-tests/support/pages/helm/controls.ts`. please confirm if  this is okay. thanks

Reply:
done, https://github.com/shvasude/console/commit/0e875a464a8fd0a089b59ee6809a182c9c859cab